src/main/scss/components/_breadcrumbs.scss

```diff
@@ -1,130 +1,91 @@
 @use "../abstracts/mixins";
 
 .jenkins-breadcrumbs {
-  position: sticky;
-  top: 0;
-  z-index: 999;
   display: flex;
   align-items: center;
-  padding: 0.55rem 0.7rem 0.55rem 0.75rem;
-  backdrop-filter: blur(15px);
-  overflow-x: auto;
-  background: var(--breadcrumbs-bar-background);
+  gap: 0.625rem;
+  margin-left: 0.625rem;
+  overflow: hidden;
 
   &__list {
     display: contents;
-    list-style-type: none;
```

Comment:
I am using the breadcrumb not in the header and I get 
![image](https://github.com/user-attachments/assets/638add27-ec8f-4973-a3a8-17551bf4d11c)
...reverting the above line 
![image](https://github.com/user-attachments/assets/73cea9c8-e058-489e-8300-a659214bc35f)
...another thing I am seeing is the last empty `li` which leads to a trailing `/` 

I am using 

```
          <l:breadcrumbBar>
                <j:out value="${breadcrumbs}" />
            </l:breadcrumbBar>
```

Reply:
The rendered output looks like 
![image](https://github.com/user-attachments/assets/a6a8875c-647d-4b49-8d66-783a22d2bb4e)

Reply:
I just updated our baseline with your latest changes and now        

```
<l:breadcrumbBar> 
  <j:out value="${breadcrumbs}" />
</l:breadcrumbBar>
```

is only rendering the `ol` and nothing more :man_shrugging:

Reply:
I am not sure why it did not work correctly the first time, but now it does
![image](https://github.com/user-attachments/assets/e5a21f26-9e18-43cf-ae4f-f8953f01fafd)